src/components/CreateCoursePage/CreateCourseForm.test.jsx

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import CreateCourseForm from './CreateCourseForm';
+import { BaseCreateCourseForm } from './CreateCourseForm';
+
+const organizations = [
+  { name: 'edX', key: 'edx' }, { name: 'edX2', key: 'edx2' },
+];
 
 describe('CreateCourseForm', () => {
   it('renders html correctly with no orgs', () => {
-    const component = shallow(<CreateCourseForm />);
+    const component = shallow(<BaseCreateCourseForm
+      handleSubmit={() => {}}
+      pristine
```

Comment:
pristine and submitting seem like they should be default props like I did here, but personal opinion https://github.com/edx/publisher-frontend/blob/5c4253d9f6e2450f0de3425b5015ef0067055d4e/src/components/EditCoursePage/EditCourseForm.jsx#L268-#L272

Reply:
I will take a look.